File: app/src/main/java/dev/ragnarok/fenrir/mvp/view/IEditPhotoAlbumView.java
package dev.ragnarok.fenrir.mvp.view;

import dev.ragnarok.fenrir.mvp.view.base.IAccountDependencyView;
import dev.ragnarok.fenrir.mvp.view.base.ISteppersView;
import dev.ragnarok.fenrir.view.steppers.impl.CreatePhotoAlbumStepsHost;

public interface IEditPhotoAlbumView extends IAccountDependencyView, ISteppersView<CreatePhotoAlbumStepsHost>, IErrorView {

}

File: app_fenrir/src/main/kotlin/dev/ragnarok/fenrir/fragment/createphotoalbum/IEditPhotoAlbumView.kt
package dev.ragnarok.fenrir.fragment.createphotoalbum

import dev.ragnarok.fenrir.fragment.base.core.IErrorView
import dev.ragnarok.fenrir.fragment.base.core.ISteppersView
import dev.ragnarok.fenrir.view.steppers.impl.CreatePhotoAlbumStepsHost

interface IEditPhotoAlbumView : ISteppersView<CreatePhotoAlbumStepsHost>,
    IErrorView